db/migrate/20200910205330_create_logs.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLogs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :logs do |t|
+      t.references :user, foreign_key: true
+      t.references :symptom, foreign_key: true
+      t.string :time
+      t.string :note
```

Comment:
The `string` datatype is limited to 255 characters. We should either make that explicit in the form instructions so the user doesn't go past that (and see if there's a way to actually validate requirement in the `#create` action in the controller)
Or if we use the `text` datatype that limit significantly increases and we don't need to worry about that.

Reply:
Yes it should be the text datatype.